models/utils.model.js

```diff
@@ -0,0 +1,11 @@
+const db = require("../db/connection");
+const format = require("pg-format");
+
+exports.checkExists = (table, column, value) => {
+  const queryString = format(`SELECT * FROM %I WHERE %I = $1;`, table, column);
```

Comment:
This is an excellent use of format for your query string to make this reusable.  Nice